src/main/java/org/jabref/gui/libraryproperties/general/GeneralPropertiesViewModel.java

```diff
@@ -141,4 +156,16 @@ public StringProperty userSpecificFileDirectoryProperty() {
     public StringProperty laTexFileDirectoryProperty() {
         return this.laTexFileDirectoryProperty;
     }
+
+    public void setUsername(String username) {
+        usernameProperty.setValue(username);
+    }
```

Comment:
This is not necessary - because the change of the username would be handled in other places in JabRef.